### DISCUSSION TITLE
iRobot Create 3: higher payload up to 15 kg

### DISCUSSION THREAD

**Discussion by AgeHelper** [iRobotEducation/create3_docs#427] [Questions] [answered]
### How are you connecting to your Create 3?

Wi-Fi (ROS 2)

### Computer(s) Model(s) and Operating System(s)

ubuntu

### Which version of ROS 2 is installed on your computer?

None

### Which firmware version is installed on your robot?

basic

### Which RMW is your robot running?

I don't know

### Is the robot connected to a network? If so, what is the network type?

yes, facility network and ioT

### Are there multiple Create 3 robots connected to your network?

None

### Is multicast enabled?

None

### What is the Adapter Board's USB/BLE Toggle currently switched to?

None

### Describe your question.

Hello, it is mentioned that for payloads of more than 9 kgs, one can build in a caster rear wheel or slow down acceleration etc.
None of these physically lower the burden of the weight on the motor, so the question is, are the 9kg specs just because other parts like the brush/wheels etc. get used up and the motor in general has no problems and can carry more? Because otherwise it`s still 15 kg the motor has to carry, whether on  2 or 3 or 4 wheels.
Big thanks in advance!!!

**Comment by shamlian** [comment, marked as answer]
There is only so much torque that the motors in the robot can provide. Additionally, you need to ensure the mass you add is over the wheels (or toward the front, in the triangle of contact between the wheels and caster), such that for the accelerations you request of the robot don't bog it down in the rear (since there's no point of contact there -- that's the purpose of the extra rear caster print). We suggest 9 kg over the front of the robot with the default acceleration profiles; if you add more weight you will likely need to lower the max acceleration to have stable control, and then at some point there will be so much weight, that the robot cannot provide enough torque (because torque is proportional to current applied to the motor) to move at all.

**Comment by AgeHelper** [comment]
Thanks a ton for the info, much appreciated 😊

 

Best,

Kailas

 

 

 

 

 

 

 

 

------------------------

Bearcover GmbH

Fließstr.18

12439 Berlin

 

Lieferungen:

Bearcover GmbH

Bouchéstraße 12/Halle 20 (MotionLab)

12435 Berlin

 

 

Von: Steven Shamlian ***@***.***> 
Gesendet: Freitag, 6. Oktober 2023 16:48
An: iRobotEducation/create3_docs ***@***.***>
Cc: AgeHelper ***@***.***>; Author ***@***.***>
Betreff: Re: [iRobotEducation/create3_docs] iRobot Create 3: higher payload up to 15 kg (Discussion #427)

 

There is only so much torque that the motors in the robot can provide. Additionally, you need to ensure the mass you add is over the wheels, such that for the accelerations you request of the robot don't bog it down in the rear (since there's no point of contact there -- that's the purpose of the extra rear caster print). We suggest 9 kg over the front of the robot with the default acceleration profiles; if you add more weight you will likely need to lower the max acceleration to have stable control, and then at some point there will be so much weight, that the robot cannot provide enough torque (because torque is proportional to current applied to the motor) to move at all.
